src/ansys/aedt/core/application/analysis.py

```diff
@@ -2402,3 +2232,273 @@ def number_with_units(self, value, units=None):
 
         """
         return self.modeler._arg_with_dim(value, units)
+
+
+class AvailableVariations(object):
+    def __init__(self, app):
+        """Contains available variations.
+
+        Parameters
+        ----------
+        app : :class:`ansys.aedt.core.application.analysis.Analysis`
+            Analysis object.
+
+        """
+        self._app = app
+        self.independent = True
+
+    @property
+    def all(self):
+        """Create a dictionary with variables names associated to ``"All"``.
+
+        Returns
+        -------
+        dict
+            Dictionary of all variables with ``"All"`` value.
+
+        """
+        return {name: "All" for name in self.__variable_names()}
+
+    @property
+    def nominal(self):
+        """Create a dictionary with variables names associated to ``"Nominal"``.
+
+        Returns
+        -------
+        dict
+            Dictionary of all variables with ``"Nominal"`` value.
+        """
+        return {name: "Nominal" for name in self.__variable_names()}
+
+    @property
+    def nominal_values(self):
+        """All variables with nominal values.
+
+        Returns
+        -------
+        dict
+            Dictionary of nominal variations with values.
+
+        """
+        available_variables = self.__available_variables()
+        return {k: v.expression for k, v in list(available_variables.items())}
+
+    @property
+    def nominal_w_values_dict(self):
+        """Nominal independent with values in a dictionary.
+
+        Returns
+        -------
+        dict
+            Dictionary of nominal independent variations with values.
+
+        References
+        ----------
+        >>> oDesign.GetChildObject('Variables').GetChildNames
+        >>> oDesign.GetVariables
+        >>> oDesign.GetVariableValue
+        >>> oDesign.GetNominalVariation
+        """
+        warnings.warn(
+            "`nominal_w_values_dict` is deprecated. " "Use `nominal_values` method instead.", DeprecationWarning
+        )
+        families = {}
+        for k, v in list(self._app.variable_manager.independent_variables.items()):
+            families[k] = v.expression
+
+        return families
```

Comment:
```suggestion
        families = {
    k: v.expression
    for k, v in self._app.variable_manager.independent_variables.items()
}

return families
```

I'd use the same approach as you did for all, nominal and nominal_values.
@Samuelopez-ansys

Reply:
This is deprecated, so no changes are needed. This will be removed in future versions.